### ISSUE TITLE
UpdateHMDMatrixPose() is slower in release than in debug mode?

### ISSUE THREAD

**Issue by ZhengzhongSun** [ValveSoftware/openvr#180]
For my program, in debug 32 mode, UpdateHMDMatrixPose() consumes nearly 0.01s, but in release 64, UpdateHMDMatrixPose() consumes nearly 0.02s and it makes my fps only 50. I'm really curious about it. Does anyone know the reason and explain it?

**Comment by JoeLudwig** [comment]
Do you mean UpdateHMDMatrixPose in the HelloVR sample?

**Comment by ZhengzhongSun** [comment]
yes. Is this the most important reason for vive helmet can not have a high fps?

**Comment by erwincoumans** [comment]
I noticed the same, UpdateHMDMatrixPose can take from 6-20ms per call, almost entirely blocked in IVRCompositor::WaitGetPoses
What is causing this?

**Comment by echuber2** [comment]
I believe we encountered the same issue after switching from an AMD system to an Nvidia system. Running a Debug x64 build is smooth but a Release x64 build has severe stuttering and frame drops, around 54fps (_plus_ the stutters) unlike 90fps as usual. I'll see if I can reproduce the issue later and narrow it down to the same function call or not.

**Comment by echuber2** [comment]
Just a follow-up. The issue was caused by vsync settings in the Nvidia control panel. We changed it from "application settings" to "off" and the fps returned to 90, stuttering gone.
